libs/auth-api-lib/src/lib/delegations/delegations.service.ts

```diff
@@ -442,15 +442,17 @@ export class DelegationsService {
    */
   private async findAllCompaniesIncoming(user: User): Promise<DelegationDTO[]> {
     try {
-      const person = await this.rskProcuringClient.getSimple(user)
+      const person = await this.rskProcuringClient.getIndividualRelationships(
```

Comment:
Need to make sure that these endpoints work without having the admin scope. Should be enough to test that procuration holder delegations still work after merging to main.